src/Metal/MTLDevice.cs

```diff
@@ -52,22 +52,34 @@ public static partial class MTLDevice {
 		}
 
 #if MONOMAC
+#if !NET
 		[NoiOS, NoWatch, NoTV]
+#endif
 		[DllImport (Constants.MetalLibrary)]
 		unsafe static extern IntPtr MTLCopyAllDevices ();
 
+#if !NET
 		[NoiOS, NoWatch, NoTV]
+#endif
```

Comment:
same.

Reply:
for a public member it could have informational value...
e.g. https://apisof.net/catalog/4b8f24e1-c4fe-17ba-d1a8-5fcaf74b3f03